data/hai/hai reveal 1 intro.txt

```diff
@@ -309,15 +307,66 @@ mission "ZZ Event Hai Secret Leaks: FW"
+			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention hey?"`
+			`	The emphasis she places on the word "gift" is dripping in sarcasm, but her overall tone is wary rather than hostile, and she gestures without comment to the seat opposite for you to sit.`
+			choice
+				`	"I'm just here to find out more about the Hai."`
+					goto innocent
+				`	"You looked to be the most experienced person in the room."`
+					goto practical
+				`	"Why ask a rookie, when there's a professional available?"`
+			`	"Pff," she scoffs, but despite rolling her eyes a slight smirk undermines her otherwise wary expression. "Flattery from a hero. You've been hanging out with too many politicians <last>."`
+			`	The deliberate omission of your title does not go unnoticed, but she continues. "Still, can't fault your logic, and I suppose I probably owe you one on account of me having got myself stuck on one of those Pug occupied worlds while it was all going on. You're going to ask me about the Hai aren't ya?"`
+			choice
+				`	"Yes."`
+					goto continue
+			label innocent
+			`	"Hah!" she booms, one deep short bark of a laugh. "You and everyone else <last>, and don't go trying to sell me the innocent question line. You didn't single me out cause I was pretty."`
+			`	She squints at you and leans back in her seat slightly. "That said, if you're asking me and not any of your new "friends" in the government... maybe you're alright in my books after all."`
+				goto continue
+			label practical
+			`	Her eyes narrow, but she dips her head in a very small motion of assent, "Well, you're not stupid, I'll give you that. You're trying to find out what most other people are aren't you? You want to know about the Hai."`
+			choice
+				`	"Yes."`
+			label continue
+			`	She heaves a sigh and lowers her voice until you're forced to lean in. "You can call me Captain Casey, and I will tell you what you want, but then you can leave alright? I like my solitude.`
+			`	"Firstly, I shouldn't be telling you this. Generally speaking they don't like to draw attention to themselves. Secondly though, forget the fearmongering. I make trips up to their space every so often and there aren't a nicer bunch of people anywhere, even if they are a bit furry. The Hai are peaceful, and welcoming of outsiders. They've got themselves a bit of an anarchist faction, that's the footage you've seen, but they ain't no danger to humanity. You should see them for yourself."`
```

Comment:
```suggestion
			`	"First, I shouldn't be telling you this. Generally speaking they don't like to draw attention to themselves. Second though, forget the fearmongering. I make trips up to their space every so often, and there aren't a nicer bunch of people anywhere, even if they are a bit furry. The Hai are peaceful and welcoming of outsiders. They've got themselves a bit of an anarchist faction, that's the footage you've seen, but they ain't no danger to humanity. You should see them for yourself."`
```

Reply:
Can you explain the logic here? I don't understand the reasoning behind any of these changes.